### ISSUE TITLE
Support for Ubuntu 16.04

### ISSUE THREAD

**Issue by sollie** [saltstack/salt-bootstrap#826]
salt-bootstrap does not currently support Ubuntu 16.04.
As there currently is no 16.04 specific version of Salt, could a temporary workaround be to default to the 14.04 build for users wanting to test out Xenial in their environments?
### Steps to Reproduce Issue

Install Ubuntu 16.04.
Download and run bootstrap.sh
### Versions and Systems

./bootstrap -- Version 2016.04.18

Last bit of output from bootstrap attempt:

```
+ return 0
+ '[' '' '!=' '' ']'
+ '[' 0 -eq 1 ']'
+ '[' '' '!=' '' ']'
+ return 0
++ egrep '^(2015\.5|2015\.8|latest|archive\/)'
++ echo latest
+ '[' latest '!=' '' ']'
++ grep -ER 'latest .+ main' /etc/apt
+ '[' '' = '' ']'
+ echo 'deb http://repo.saltstack.com/apt/ubuntu/16.04/amd64/latest trusty main'
+ __apt_get_install_noinput wget
+ apt-get install -y -o DPkg::Options::=--force-confold wget
Reading package lists...
Building dependency tree...
Reading state information...
wget is already the newest version (1.17.1-1ubuntu1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
+ return 0
+ apt-key add -
+ wget -q https://repo.saltstack.com/apt/ubuntu/16.04/amd64/latest/SALTSTACK-GPG-KEY.pub -O -
gpg: no valid OpenPGP data found.
+ return 1
+ '[' 1 -ne 0 ']'
+ echoerror 'Failed to run install_ubuntu_stable_deps()!!!'
+ printf '\033[1;31m * ERROR\033[0m: %s\n' 'Failed to run install_ubuntu_stable_deps()!!!'
 * ERROR: Failed to run install_ubuntu_stable_deps()!!!
+ exit 1
+ __exit_cleanup
+ EXIT_CODE=1
+ '[' stable = git ']'
+ echodebug 'Removing the logging pipe /tmp/bootstrap-salt.logpipe'
+ '[' 0 -eq 1 ']'
+ rm -f /tmp/bootstrap-salt.logpipe
++ awk '{print $1}'
++ grep /tmp/bootstrap-salt.log
++ grep tee
++ ps ax

```

**Comment by dmurphy18** [comment]
@sollie Ubuntu 16.04 is scheduled to be released today.  SaltStack has built salt 2015.8.8 for Ubuntu 16.04 Beta 2 (which actually shipped with Salt version 2015.8.7 as part of the ISO).  We shall have a version of Salt available for Ubuntu 16.04 LTS in the next point release which should be quite soon, once SaltConf finishes today.

**Comment by cnk** [comment]
@dmurphy18 Fabulous. Looking forward to salt-bootstrap working for the new 16.04 LTS release. Thanks for maintaining this service.

**Comment by strzelecki-maciek** [comment]
anxiously waiting for the 16.04 fix too ;-)

**Comment by sollie** [comment]
@dmurphy18 I'm just so used to having old versions shipped with the OS that I never checked what they had packaged for 16.04, that's my bad. Happy to report that I just installed 2015.8.8 from the official ubuntu repos.

It would still be nice eventually having support for it in salt-bootstrap for compatibility with existing deployment setups.

**Comment by rallytime** [comment]
@sollie This should be fixed by the changes in PR #852, which was just merged into the develop branch. I'll close this for now, but if anything pops up, let us know and we'll be happy to take another look.
